contracts/rollup.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
@otherview Can you add a readme to this folder, describing what this is for and where it's used? I couldn't work it out.

Reply:
Added 👍